src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -209,22 +223,27 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 				String formDetails = sb.toString();
 				String tokenNamesFlattened = tokenNames.toString();
 				
+				int risk = Alert.RISK_LOW;
 				String desc = Constant.messages.getString("pscanbeta.noanticsrftokens.desc");
 				String extraInfo = Constant.messages.getString("pscanbeta.noanticsrftokens.alert.extrainfo", tokenNamesFlattened, formDetails);
+				if (hasSecurityAnnotation) {
+					risk = Alert.RISK_INFO;
+					extraInfo = Constant.messages.getString("pscanbeta.noanticsrftokens.extrainfo.annotation");
+				}
 				
-			    Alert alert = new Alert(getPluginId(), Alert.RISK_LOW, Alert.CONFIDENCE_MEDIUM,  getName());
-			    alert.setDetail(
-			    			desc + "\n"+getDescription(), 
-				    		msg.getRequestHeader().getURI().toString(),
-				    		"",  //parameter: none.
-				    		"", 
-				    		extraInfo,
-				    		getSolution(), 
-				            getReference(), 
+				Alert alert = new Alert(getPluginId(), risk, Alert.CONFIDENCE_MEDIUM,  getName());
```

Comment:
Seems to be a bunch of formatting changes here.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting